test/Engine/ProtoTest/FFITests/CSFFITest.cs

```diff
@@ -399,6 +399,7 @@ public void TestInheritanceCtorsVirtualMethods3()
         }
 
         [Test]
+        [Category("DSDefinedClass")]

```

Comment:
This is the diff tool just conking out. The only changes here are adding cateogries